lib/reel/websocket.rb

```diff
@@ -10,10 +9,11 @@ class WebSocket
 
     attr_reader :socket
     def_delegators :@socket, :addr, :peeraddr
+    def_delegators :@driver, :ping
 
     def initialize(info, connection)
-      driver_env = DriverEnvironment.new(info, connection.socket)      
-      
+      driver_env = DriverEnvironment.new(info, connection.socket)
+
       @socket = connection.hijack_socket
       @request_info = info
 
```

Comment:
see the comments in the original PR. `Driver.server` calls `Driver.rack` eventually, after parsing the headers on it's own. websocket-driver does not depend on rack itself, but is built to expect rack apparently.